backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/nudger/BpkNudger.kt

```diff
@@ -0,0 +1,91 @@
+/*
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.compose.nudger
+
+import androidx.compose.animation.animateColorAsState
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import net.skyscanner.backpack.compose.button.BpkButton
+import net.skyscanner.backpack.compose.button.BpkButtonSize
+import net.skyscanner.backpack.compose.button.BpkButtonType
+import net.skyscanner.backpack.compose.fieldset.BpkFieldStatus
+import net.skyscanner.backpack.compose.fieldset.LocalFieldStatus
+import net.skyscanner.backpack.compose.icon.BpkIcon
+import net.skyscanner.backpack.compose.text.BpkText
+import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+import net.skyscanner.backpack.compose.tokens.Minus
+import net.skyscanner.backpack.compose.tokens.Plus
+
+@Composable
+fun BpkNudger(
+  value: Int,
+  onValueChange: (Int) -> Unit,
+  range: IntRange,
+  modifier: Modifier = Modifier,
+  enabled: Boolean = LocalFieldStatus.current != BpkFieldStatus.Disabled,
+) {
+
+  val trimmedValue = value.coerceIn(range)
+
+  Row(
+    modifier = modifier,
+    verticalAlignment = Alignment.CenterVertically,
+  ) {
+
+    BpkButton(
+      icon = BpkIcon.Minus,
+      contentDescription = "-",
```

Comment:
For the view version we did this to cheat and add accessibility without making a breaking change in the API. Now that we're starting from fresh let's add a parameter for this. ie web has a `decreaseButtonLabel` and `increaseButtonLabel`
Alternatively I wonder if we can re-use the progress semantics on the whole component to allow increasing it from the slider. but it might be that that only supports percentages, rather than absolute values (similar to the view version)

We also need to announce the state when the value is increased/decreased

Reply:
+/- is still a bit of a hack and I'm not sure if it works great in all languages either. 

Slider is one option (that's what iOS uses) - the other is simply announcing the state after changing it which the current nudger component does as well. We definitely need to announce _something_, though.